### DISCUSSION TITLE
ECE%40UT%20Austin/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#219] [Announcements]
# ECE%40UT%20Austin/

None

https://opencs.app/ECE%40UT%20Austin/

**Comment by yuhengcai1** [comment]
如果写了想继续读PHD的话会直接按照PHD标准发offer（指直接发PHD的offer），如果只想申请硕士的同学注意不要写

**Reply by QAQdev** [reply]
这个有什么影响吗
